fastlane/Fastfile

```diff
@@ -20,12 +20,12 @@ default_platform(:ios)
 platform :ios do
   desc "Run Tests and Output Code Coverage"
   lane :test do |options|
-    workspace = options[:workspaceFilename]
+    workspace = "OneLogin.xcworkspace"
 
     clear_derived_data
       
     run_tests(
-      workspace: workspace,
+      workspace: "OneLogin.xcworkspace",
```

Comment:
same thing here, this could have been missed but need to pass this from the action so should be `options[:workspaceFilename]`

Reply:
[Ibid](https://github.com/alphagov/di-mobile-ios-onelogin-app/pull/1#discussion_r1312772201)